### DISCUSSION TITLE
自訂義基建換幹員錯誤求助(繁中服)

### DISCUSSION THREAD

**Discussion by hubertka997799** [MaaAssistantArknights/MaaAssistantArknights#4575] [Q&A]
問題狀況:3班制換班時有某一時段的控制中樞只會出現一個幹員(僅此時段)

設定&環境:我使用的是在一圖流網站自訂的243換班、藍疊Hyper-V版

問題複現:
前一班:["阿米娅","凯尔希","琴柳","令","夕"] 切換到後一班 ["诗怀雅","凯尔希","焰尾","老鲤","灵知"] 有高機率出現這個問題
MAA運行紀錄會出現後一班的五人名子，首次辨識成功選擇五人後點選排序並拉回最左邊後，本應重新選取這五人，但靈知不會被選到，然後MAA就開始往後滑重新尋找，尋找無果後運行紀錄會只顯示靈知並且儘選取靈知進入

嘗試的方向:
靈知改填入繁中名=>連首次辨識都無法
手動再按一次換班(也就是從中樞僅有靈知的情況下)=>會正常換班
MAA重新尋找往右滑的階段強行往左拉回到靈知的位置=>有機會成功選取靈知

**Comment by ABA2396** [comment]
去  [这里](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new?assignees=&labels=bug&template=cn-bug-report.yaml) 发下 issue 并带上日志文件
